hadoop-tools/hadoop-sls/src/main/java/org/apache/hadoop/yarn/sls/scheduler/SLSCapacityScheduler.java

```diff
@@ -105,7 +108,11 @@ public boolean tryCommit(Resource cluster, ResourceCommitRequest r,
 
   @Override
   public void handle(SchedulerEvent schedulerEvent) {
-    schedulerCommons.handle(schedulerEvent);
+    try{
+      schedulerCommons.handle(schedulerEvent);
```

Comment:
This catch can be problematic, cause there is a chance that some logic what used this code expected an exception in case of the SLSCapacityScheduler can not handle the event, what exception now wont be provided for the caller.
Please can you check how this method is used by the rest of the code?

Reply:
There are more than 100 instances where this handle method is being called, I have checked fewof them and updated the code.